android/src/main/java/com/tailscale/ipn/ui/view/SettingsView.kt

```diff
@@ -45,81 +44,104 @@ fun SettingsView(
   val handler = LocalUriHandler.current
   val user = viewModel.loggedInUser.collectAsState().value
   val isAdmin = viewModel.isAdmin.collectAsState().value
-  val settingsBundles = viewModel.settings.collectAsState().value
+  val managedBy = viewModel.managedBy.collectAsState().value
 
   Scaffold(
       topBar = { Header(title = R.string.settings_title, onBack = settingsNav.onBackPressed) }) {
           innerPadding ->
-        LazyColumn(modifier = Modifier.padding(innerPadding)) {
-          item {
-            UserView(
-                profile = user,
-                actionState = UserActionState.NAV,
-                onClick = viewModel.navigation.onNavigateToUserSwitcher)
-          }
+        Column(modifier = Modifier.padding(innerPadding)) {
+          UserView(
+              profile = user,
+              actionState = UserActionState.NAV,
+              onClick = viewModel.navigation.onNavigateToUserSwitcher)
 
           if (isAdmin) {
-            item {
-              Spacer(modifier = Modifier.height(4.dp))
-              AdminTextView { handler.openUri(Links.ADMIN_URL) }
-            }
+            Spacer(modifier = Modifier.height(4.dp))
+            AdminTextView { handler.openUri(Links.ADMIN_URL) }
           }
 
-          settingsBundles.forEach { bundle ->
```

Comment:
Making this a bit more literal reduces the number of StateFlows that we have to create and is IMHO more readable.

Reply:
DNS and tailnetLock will end up getting wrapped in let? block too (they can be disabled via MDM), so it's a state flow per setting, instead of a stateflow for all of the settings... But this does look cleaner.